templates/motor-insurance/layout/summary.js

```diff
@@ -0,0 +1,201 @@
+import { html, classMap, templates } from '/src/main.js';
```

Comment:
This file is getting rather large. We may need to do some work on summaries in the future.

Reply:
It was explicitly requested (by Marcus I believe) to inline all the summary bits into `layout/summary.js` (rather than keep using `shared/summary.js` for UI wrappers)

I guess the objective is to make it easier for a user to reason about by keeping all the functions close (not sure about that tbh).

No objections on some polishing later, but we'll need to do that for all the domains, so it sounds like a separate piece of work once we're happy with each domain's templates in general

Reply:
Yup, this is something for consideration later.